laelf.adoc

```diff
@@ -731,89 +731,89 @@ See <<code_models>> for how it works on various code models.
 
 |110
 |R_LARCH_CALL36
-|Used for medium code model function call sequence `pcaddu18i + jirl`. The two instructions must be adjacent.
+|Used for medium code model function call sequence `pcaddu18i + jirl`. The two instructions must be adjacent. `%call36(symbol)`
```

Comment:
Is this better? It's okay either way to me because this one description's pattern is different than the others.

```suggestion
|Used for medium code model function call sequence `pcaddu18i + jirl`, `%call36(symbol)`. The two instructions must be adjacent.
```